net.sourceforge.GTKWave.yml

```diff
@@ -0,0 +1,47 @@
+app-id: net.sourceforge.GTKWave
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: gtkwave
+rename-icon: gtkwave
+rename-desktop-file: gtkwave.desktop
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+modules:
+  - name: tcl
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/tcl/Tcl/8.6.12/tcl8.6.12-src.tar.gz
+        sha256: 26c995dd0f167e48b11961d891ee555f680c175f7173ff8cb829f4ebcde4c1a6
+    subdir: "unix"
+    post-install:
+      - chmod 755 /app/lib/libtcl*.so
+  - name: tk
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/tcl/Tcl/8.6.12/tk8.6.12-src.tar.gz
+        sha256: 12395c1f3fcb6bed2938689f797ea3cdf41ed5cb6c4766eec8ac949560310630
+    subdir: "unix"
+    post-install:
+      - chmod 755 /app/lib/libtk*.so
+  - name: gtkwave
+    config-opts:
+      - --enable-gtk3
+      - --enable-manymarkers
+      - --disable-dependency-tracking
+      - --disable-mime-update
+    subdir: "gtkwave3-gtk3"
+    post-install:
+      - install -Dm644 share/appdata/gtkwave.appdata.xml /app/share/metainfo/net.sourceforge.GTKWave.metainfo.xml
+    sources:
+      - type: git
+        url: http://github.com/gtkwave/gtkwave
+        branch: master
```

Comment:
this needs to be a specific commit rather than just a branch that can keep changing. other than that lgtm